overlord/servicestate/quota_control_test.go

```diff
@@ -127,6 +128,7 @@ func checkQuotaState(c *C, st *state.State, exp map[string]quotaGroupState) {
 }
 
 func checkSvcAndSliceState(c *C, snapSvc string, sliceName string, sliceMem quantity.Size) {
```

Comment:
should the input be called slicePath then?

Reply:
done